data/attributes.yml

```diff
@@ -30209,6 +30209,36 @@ base_attributes:
   values:
   - vehicle_fluid_type__concentrate
   - vehicle_fluid_type__prediluted
+- id: 3415
+  name: Caffeine content
+  description: Indicates the amount of caffeine present in the coffee product, aiding consumers in choosing based on their desired caffeine intake level
+  friendly_id: caffeine_content
+  handle: caffeine-content
+  values:
+  - vcaffeine_content__regular
+  - caffeine_content__decaf
+- id: 3416
+  name: Grind size
+  description: Refers to the coarseness or fineness of the coffee grounds, influencing the extraction process and flavor profile of the brewed coffee
```

Comment:
This should be more consistent with previous entries, e.g.:
"Indicates the coarseness or fineness of coffee grounds, suitable for various brewing methods, such as extra coarse or fine"